src/sentry/scim/endpoints/utils.py

```diff
@@ -100,9 +134,9 @@ def parse_filter_conditions(raw_filters):
     # TODO: support "and" operator
     # TODO: support email querying/filtering
     # TODO: graceful error handling when unsupported operators are used
-    filters = []
+
     if raw_filters is None:
-        return filters
+        return None
```

Comment:
this is some leftover crud from how I previously wrote the function and then subsequently rewrote (used to return list, now a singular value) -- None is the correct return type and callers were doing basic `if` checks so no behavior change. This is something typing would have caught (typed SCIM routes coming soon).